client.go

```diff
@@ -84,7 +84,7 @@ func (myClient *Client) ParseCommand(firstLine []byte, isMultiplexing bool) (res
 		protocol.Debug("Received error from GetCommand: %s\r\n", err)
 		return false, err
 	}
-//	protocol.Debug("Received %s %s\r\n", myClient.command[0:myClient.commandLength], myClient.firstArgument)
+	protocol.Debug("Received %s %s\r\n", myClient.command[0:myClient.commandLength], myClient.firstArgument)
```

Comment:
`go tool objdump` shows
```
client.go:87    0x8d1ce e8cd820900          CALL github.com/forcedotcom/rmux/protocol.Debug(SB)
```

So looks like we should remove this line. I wonder if there's a way to remove function calls like this unless certain build flags are enabled...